code/src/App.js

```diff
@@ -1,9 +1,64 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react';
+
+import Header from 'components/Header';
+import ThoughtInput from 'components/ThoughtInput';
+import { ThoughtList } from './components/ThoughtList';
+import { THOUGHTS_URL } from './urls';
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([]);
+
+  useEffect(() => {
+    fetchThoughts();
+  }, []);
+
+  const fetchThoughts = () => {
+    fetch(THOUGHTS_URL)
+      .then(res => res.json())
+      .then(data => setThoughts(data))
+      .catch(error => console.error(error));
+  }
+
+  const postThought = (newThought) => {
+    fetch(THOUGHTS_URL, { 
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ message: newThought})
+  })
+    .then(() => fetchThoughts())
+    .catch(error => console.error(error));
+}
+
+const postHearts = (messageId) => {
+  const LIKES_URL = `https://happy-thoughts-technigo.herokuapp.com/thoughts/${messageId}/like`;
```

Comment:
You can save basis of URL in const and when you need it you can just write the const and add the part who is missing. It's more readable. 
example:
const baseAPI = 'https://happy-thoughts-technigo.herokuapp.com'

and in line 33 you can just do like this:

const LIKES_URL= `${baseAPI}/thoughts/${messageId}/like`